src/ui/HorizontalContentCardConnected/horizontalContentCardComponentMapper.js

```diff
@@ -1,121 +0,0 @@
-import React from 'react';
```

Comment:
We're going to need to bring these back too. Mostly because these files handle showing images on the tiles (think sermons) and the custom card labels ("1 of 7") and such.

Reply:
yep

Reply:
I think we're good now without this.